src/callbacks.rs

```diff
@@ -163,36 +177,87 @@ pub extern fn view_request_geometry(_view: WlcView, _geometry: &Geometry) {
 }
 
 pub extern fn pointer_button(view: WlcView, _time: u32,
-                         mods: &KeyboardModifiers, _button: u32,
+                         mods: &KeyboardModifiers, button: u32,
                              state: ButtonState, point: &Point) -> bool {
-    if state == ButtonState::Pressed && !view.is_root() {
-        if let Ok(mut tree) = try_lock_tree() {
-            tree.set_active_view(view)
-                .unwrap_or_else(|err| {
-                    match err {
-                        TreeError::ViewNotFound(handle) => {
-                            if handle.get_class() != "Background" {
-                                error!("Could not set active container {:?}", err);
-                            }
-                        },
-                        err => error!("Could not set active container {:?}", err)
-                    }
-                });
+    if state == ButtonState::Pressed {
+        if button == 0x110 && !view.is_root() {
+            if let Ok(mut tree) = try_lock_tree() {
+                tree.set_active_view(view)
+                    .unwrap_or_else(|err| {
+                        error!("Could not set active container {:?}", err);
```

Comment:
Ok, I'll leave the logging in but add a check so it doesn't print for the background